pkg/policytools/policytools.go

```diff
@@ -29,6 +30,7 @@ environments managed by Advanced Cluster Management.`,
 		policyCmd, // policytools
 		// Subcommands
 		&templateresolver.TemplateResolver{}, // template-resolver
+		&dryrun.DryRunner{},
```

Comment:
(nit) It's a silly thing, but just to follow the convention I've started:

```suggestion
		&templateresolver.TemplateResolver{}, // template-resolver
		&dryrun.DryRunner{},                  // dryrun
```